scripts/test

```diff
@@ -11,7 +11,7 @@ if [ -z $GITHUB_ACTIONS ]; then
     scripts/check
 fi
 
-${PREFIX}pytest $@
+${PREFIX}coverage run --debug config -m pytest
```

Comment:
I don't think we want the `--debug config` here?...

```suggestion
${PREFIX}coverage run -m pytest
```

Reply:
Well, yes, it's a bit verbose